### PR TITLE
Add "Elm online tools" draft

### PR DESCRIPTION
**Pull request by marc136**
As mentioned in [discord](https://discord.com/channels/810480359791132702/943872516261290004/1288448484474880000) I started writing up a couple of browser-based tools.

Not sure yet where to place it, and I might try to add more soonish.

### CODE REVIEW

**Comment by jfmengels** [comment]
There's also https://html-to-elm.com/
I'll add more comments if I can think of more.

**Review by supermario — APPROVED** [review]
A few small suggestions but otherwise LGTM 👍

File: content/build/online-tools.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## Literate/Notebook programming
+Where you write a (text) document, and can have interactive code segments inside.
+
+* [elm-notebook.org](https://elm-notebook.org/) comes with user management and allows to store private and public notebooks.
+
+
+## Code Generation
+
+* With Alex Korban's [json2elm](https://korban.net/elm/json2elm/) you can paste a JSON sample and it will generate decoders and encoders for you. You can switch between different styles and packages.
```
[the remark this answers is not in the document]
Comment (marc136) [reply content/build/online-tools.md:32]: I thought it would be nice if they were not all uniform :sweat_smile:, but consistency is a good goal.

**Comment by marc136** [comment]
I wanted to add Jeroens link, and I think there were also tools to convert svg to elm, but I didn't look for them yet.
Maybe I have time this evening.

**Comment by supermario** [comment]
Okay sure. Note that [rough and early content, draft or notes](https://elmcraft.org/about/markers) grade contributions with further edits later are totally acceptable :)

**Comment by marc136** [comment]
> Okay sure. Note that [rough and early content, draft or notes](https://elmcraft.org/about/markers) grade contributions with further edits later are totally acceptable :)

OK, ready now.
I didn't find the other tools I was thinking about, but Dillon's tool also can do svg.

**Review by supermario — APPROVED** [review]
Beautiful, thanks @marc136 !

**Pull request merged**